locales/nn-NO/src/studio.ts

```diff
@@ -383,6 +383,8 @@ export default removeUndefinedLocaleResources({
   /** Error text shown when form is unable to find an array item at a given keyed path */
   'form.error.no-array-item-at-key':
     'Ingen element i lista med `_key` <code>"{{key}}"</code> funne på sti <code>{{path}}</code>',
+  /** Form field deprecated label */
+  'form.field.deprecated-label': 'forelda',
```

Comment:
@s-rd I think this needs to be changed - at the very least it should be capitalized. In the bokmål translation we chose "Utfaset" here. Thoughts?